### PR TITLE
BAU: Miscellaneous small fixes

### PR DESCRIPTION
**Pull request by CharlesIC**
-  Remove dupe AWS dependency
-  Include all ts and js files in linting
-  Organise resource files
-  Format tests and fix invalid mobile validator tests
   The invalid numbers test wasn't actually checking anything. Fix it to run the code and do correct checks.
   Remove an email validator test as the `copyAndPaste@_some.domain` email address actually conforms to RFC822.

### CODE REVIEW

File: express/tests/middleware/emailValidator.test.ts [hunk 1]
```diff
@@ -1,37 +1,46 @@
...
+        );
+    });
 
     it(`should return false for the email address ${EMAIL_WITH_NO_AT_SYMBOL}`, async () => {
-        assert.equal(isRfc822Compliant(EMAIL_WITH_NO_AT_SYMBOL), false, `Expected ${EMAIL_COMPLYING_WITH_RFC822} to be rejected as invalid`);
-    })
+        assert.equal(
+            isRfc822Compliant(EMAIL_WITH_NO_AT_SYMBOL),
+            false,
+            `Expected ${EMAIL_COMPLYING_WITH_RFC822} to be rejected as invalid`
```
Comment (JohnWattsGDS) [inline express/tests/middleware/emailValidator.test.ts:35]: Should this be: `EMAIL_WITH_NO_AT_SYMBOL`?

Comment (CharlesIC) [reply express/tests/middleware/emailValidator.test.ts:35]: Probably, I just applied formatting but it's possible/likely there are bugs in the tests.

Comment (CharlesIC) [reply express/tests/middleware/emailValidator.test.ts]: OK I fixed this for the sake of it but these tests need rewriting anyway

**Review by HarshVasudev — APPROVED** [review]
Looks okay.

**Review by HarshVasudev — APPROVED** [review]

**Pull request merged**